next-app/src/components/common/SideNavBar/MenuItem/MenuItem.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import Image from 'next/image'
+
+import * as S from './MenuItem.style';
+
+export const MenuItem = ({
+  iconUrl,
+  title,
+}: {
+  iconUrl: string
+  title: string
+}) => {
```

Comment:
```suggestion
import * as S from './MenuItem.style';

interface Props {
  iconUrl: string;
  title: string;
}

export const MenuItem = ({
  iconUrl,
  title,
}: Props) => {
```
props 타입은 이런 형태로 작성해주세요!

Reply:
넵 interface로 선언해서 사용할게요!